styles/globals.css

```diff
@@ -14,3 +15,83 @@ a {
 * {
   box-sizing: border-box;
 }
+
+/* add css for table responsive */
+
+/* 
+Max width before this PARTICULAR table gets nasty
+This query will take effect for any screen smaller than 760px
+and also iPads specifically.
+*/
+@media only screen and (max-width: 760px),
+  (min-device-width: 768px) and (max-device-width: 1024px) {
+  .square-grid {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .responsivebar {
```

Comment:
I think the css settings for `responsivebar` still needs some work @Amutha37...

this is a screenshot of the progress bar in see from my mobile device (Pixel4a) - see how the progress bar isn't fitting in screen?
![image](https://user-images.githubusercontent.com/4408259/134006521-13924670-550f-4fcf-b087-3fcdeda5e673.png)

Reply:
Yes, did try to work on the svg bar. It's acting funny I will try to work on it. Anyone else want to have a go on the svg bar try it too.